wiki/Tournaments/CMC/2020/en.md

```diff
@@ -0,0 +1,388 @@
+  - [LV.4 - Angel dust (Fontes) \[4k leqek's Hard\]](https://osu.ppy.sh/beatmapsets/970810#mania/2039046)
+  - [Ben Briggs - Fearless Living (RuleBlazing) \[Coward\]](https://osu.ppy.sh/beatmapsets/1283930#mania/2666220)
+  - [Thaehan - Final Boss (Theresa May) \[Stage 4 : Credits\]](https://osu.ppy.sh/beatmapsets/980090#mania/2051097)   
+
+### Qualifiers
+
+**[Download the mappack here! (22 MB)](https://mega.nz/file/3ZJBzQ6Y#_zFdkYDEkMnSckC9bx_Sl_0358CHZU_ZWOEPtfz2yvI)**
+
+- FreeMod
+  - [Shawn Wasabi + YDG - Burnt Rice (HowToPlayLN) \[Overcooked\]](https://osu.ppy.sh/beatmapsets/995871#mania/2083163)
+  - [Yuzuki - Dear You (Vocal Version) (RemFangirl) \[Distant\]](https://osu.ppy.sh/beatmapsets/763326#mania/1605032)
+  - [USAO - Knight Rider (Spy) \[Mashiro's NORMAL\]](https://osu.ppy.sh/beatmapsets/1007938#mania/2138728)
+  - [xi - Titania (Murumoo) \[Lirai's EXPERT\]](https://osu.ppy.sh/beatmapsets/1174746#mania/2487983)
+  - [DETRO - volcanic (\[Crz\]Crysarlene) \[wrong one\]](https://osu.ppy.sh/beatmapsets/984123#mania/2059102)  
+
+## Match results
+
+### Finals week 2
+
+Saturday, 19 December 2020:
+
+| Bracket | Team 1 |  |  | Team 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Lower | UwU Team | 2 | **7** | **Leviathan** | [#1](https://osu.ppy.sh/community/matches/71838770) |
+
+Sunday, 13 December 2020, Grand Final:
+
+| Bracket | Team 1 |  |  | Team 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Grand Final | **Neppu-Neppu** | **7** | 2 | Leviathan | [#1](https://osu.ppy.sh/community/matches/71903519) |
+
+### Finals week 1
+
+Saturday, 12 December 2020:
+
+| Bracket | Team 1 |  |  | Team 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Upper | UwU Team | 0 | **7** | **Neppu-Neppu** | [#1](https://osu.ppy.sh/community/matches/71412828) |
+
+Sunday, 13 December 2020:
+
+| Bracket | Team 1 |  |  | Team 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Lower | **Leviathan** | **7** | 3 | Team 11 | [#1](https://osu.ppy.sh/community/matches/71475447) |
+
+### Semifinals
+
+Friday, 4 December 2020:
+
+| Bracket | Team 1 |  |  | Team 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Lower | Team 3 | 2 | **6** | Leviathan | [#1](https://osu.ppy.sh/community/matches/70941169) |
+
+Saturday, 5 December 2020:
+
+| Bracket | Team 1 |  |  | Team 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Lower | L Hunter | 2 | **6** | **Team 11** | [#1](https://osu.ppy.sh/community/matches/70998868) |
+| Upper | SCP Indonesia | 2 | **6** | **UwU Team** | [#1](https://osu.ppy.sh/community/matches/71003960) |
+| Upper | **Neppu-Neppu** | **6** | 3 | GWS | [#1](https://osu.ppy.sh/community/matches/71008403) |
+
+Sunday, 6 December 2020:
+
+| Bracket | Team 1 |  |  | Team 2 | Match link |
+| :-: | --: | :-: | :-: | :-- | :-- |
+| Lower | GWS | 2 | **6** | Team 11 | [#1](https://osu.ppy.sh/community/matches/71071493) |
+| Lower | SCP Indonesia | 0 | **6** | **Leviathan** | *win by default* |
```

Comment:
should "w" be capitalised here? and should "win" be past tense?

Reply:
this is actually how it is supposed to be written. even in other tournament articles (including the World Cup pages) the "*win by default*" here has been always left as is to properly indicate the match output

![image](https://user-images.githubusercontent.com/36564236/103252171-378c2280-49ae-11eb-90de-95056a1c65fb.png)